docker-compose.yml

```
@@ -30,7 +30,8 @@ talkapi:
     - redis
 
 sidekiq:
-  image: talk-api:local
+  dockerfile: Dockerfile
+  build: ./
```

It would be best to reuse the image from talkapi
```suggestion
  image: talk-api:local
```

and change the  talkapi build context to include the local image name, https://docs.docker.com/compose/compose-file/#build
```
talkapi:
  dockerfile: Dockerfile
  build: ./
  image: talk-api:local
````

---

Thanks to Docker's layer caching it doesn't actually build anything the second time.